index.js

```diff
@@ -69,10 +77,15 @@ async function deleteOlderReleases(keepLatest) {
   try {
     let data = await fetch({
       ...commonOpts,
-      path: `/repos/${owner}/${repo}/releases`,
+      path: `/repos/${owner}/${repo}/releases?per_page=100`,
```

Comment:
Any reason you mentioned `per_page=100` ?

Reply:
default is 30, maximum is 100. I didn't want to add full pagination support too. My project hat ~170 Releases and i Wanted to keep al the production Releases (around 25) so there were only 5 spots per execution for prereleases to be deleted.

My strategy now is to keep 10 of the releases containing 'test' and 70 releases in total. That way I can see the Releases of the last ~year (60 weeks, I try to release one a week)

For this Strategy to work with "per_page=100" I can have 20 releases to be deleted between clean up runs.